w2c2/w2c2_base.h

```diff
@@ -52,6 +52,12 @@ typedef double F64;
 #define W2C2_LL(x) x ## ll
 #endif
 
+/* Prevent LLVM/Clang <=20 from optimizing away infinite loops */
+#if defined(__clang__) && __clang_major__ <= 20
```

Comment:
Versions <12 don't seem to be affected. Versions >20 will likely keep being affected.

```suggestion
/* Prevent LLVM/Clang >=12 from optimizing away infinite loops */
#if defined(__clang__) && __clang_major__ >= 12
```

Reply:
Never mind, looked at your comments. I'll update